### PR TITLE
Add RC Link Preset section

### PR DESCRIPTION
**Pull request by hyp0dermik-code**
Addresses #379

### CODE REVIEW

**Comment by hyp0dermik-code** [comment]
While I agree with @VitroidFPV that this is best for the Betaflight wiki, I think there would need to be some structure changes which might result in the link changing which is beyond the current scope.

As such I suspect it would be best to have it here for the moment, and perhaps look at moving it to the BF wiki when the structure has been tidied up a bit.

**Comment by hyp0dermik-code** [comment]
Open for feedback on the concerns location in particular, along with any other suggestions

File: docs/quick-start/pre-1stflight.md [hunk 1]
```diff
@@ -86,6 +86,29 @@ On the 900MHz hardware, a value of -20dBm or thereabouts is a good indication yo
 - Most DIY modules require moving the 0-ohm resistor on the E28 from the PCB antenna side towards the UFL side. A solder bridge will work just fine as well, but make sure it is on the correct pads.
 - Swap out antennas on the receiver and/or the TX module; Most ExpressLRS receiver antennas use IPEX 1/UFL connectors and as long as the frequency tuning of the antenna is appropriate for the frequency you're in, it should work. You can also use old 2.4GHz WiFi Router antennas for your 2.4GHz modules, but avoid those that are dual-band. Also, make sure that the connector on the antenna is the appropriate one (RP-SMA on R9 modules; SMA on most off-the-shelf ExpressLRS modules)
 
+## RC Link Preset
+For flight controllers based on Betaflight, there are 'RC Link Presets' available that configure feedforward smoothing and associated link settings based on your packet rate and usage case. Using no link preset, or using the incorrect one for your packet rate, can result in unwanted disturbances in feedforward, which can affect setpoint tracking.
```
Comment (deadbytefpv) [inline docs/quick-start/pre-1stflight.md:90]: > can result in unwanted disturbances in feedforward

would be great if we can be a bit specific. e.g. "can result in jittery or noisy signal"

**Review by deadbytefpv — CHANGES_REQUESTED** [review]

File: docs/quick-start/pre-1stflight.md (hunk 1 above)
Comment (deadbytefpv) [reply docs/quick-start/pre-1stflight.md:90]: you'll have to add a few words in the wordlist to appease the spellchecker or use different words altogether.

**Review by deadbytefpv — DISMISSED** [review]

**Comment by maybenikhil** [comment]
@hyp0dermik-code moved this to fc configuration (opinionated change) because I feel like it flows with the section better, and given the user has technically already read lua config in terms of the flow of the guide, they should know what frequency they wanted to use. Also changed some verbiage to be more accessible, particularly "next highest" felt unclear, especially to an audience that isn't always english fluent

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**